cms/djangoapps/contentstore/management/commands/populate_creators.py

```diff
@@ -1,60 +0,0 @@
-"""
```

Comment:
This management command seems like a temporary migration to update course access.